lib/grammers-client/src/client/channel.rs

```diff
@@ -0,0 +1,75 @@
+use super::Client;
+use anyhow::{anyhow, Result};
+use grammers_tl_types::{
+    enums::{
+        channels::ChannelParticipants::{NotModified, Participants},
+        messages::ChatFull,
+        ChannelParticipantsFilter,
+        InputChannel::Channel,
+        User,
+    },
+    functions::channels::{GetFullChannel, GetParticipants},
+    types::InputChannel,
+};
+
+impl Client {
+    const MAX_PARTICIPANT_LIMIT: usize = 200;
+
+    pub async fn get_full_channel(&mut self, channel_id: i64) -> Result<ChatFull> {
```

Comment:
Instead of the `channel_id`, this should use the same `PackedChat` as input, like the other client methods do. See for example [`send_message`](https://github.com/Lonami/grammers/blob/3ed47232c01dc3c05426cc3c346e7deb1dadc002/lib/grammers-client/src/client/messages.rs#L381-L385) to know what the signature should look like.